src/main/resources/feedbackQuestionTextEditFormTemplate.html

```diff
@@ -0,0 +1,17 @@
+<div>
+    <br>
+    <div class="row">
+        <div class="col-xs-11">
+            <label class="float-left padding-top-7 text-color-gray font-weight-normal margin-right-10">[Optional]</label>
+            <label class="float-left padding-top-7 text-color-gray font-weight-normal margin-right-10 border-bottom-dotted"
+                    data-toggle="tooltip" data-placement="top" title="the recommended length is shown to the respondent but not enforced">Recommended
+                    length</label>
+            <label class="float-left padding-top-7 text-color-gray font-weight-normal">for the answer:</label>
+            <div class="col-xs-3 width-80-px padding-0 margin-right-10">
+                <input disabled type="number" class="form-control" name="recommendedlength" value="${recommendedlength}">
+            </div>
+            <label class="padding-top-7 text-color-gray font-weight-normal">words<label>
```

Comment:
Since the whole line of text is conceptually one "thing", is it possible to not break it into so many labels. Perhaps something like
```
<div class="question-recommended-length">
    ... <span class="underline-dotted">Recommended length</span> ... <input> ...
</div>
```